test/polyfills.js

```diff
@@ -3,3 +3,11 @@ import 'core-js/es6/map';
 import 'core-js/fn/array/fill';
 import 'core-js/fn/array/from';
 import 'core-js/fn/object/assign';
+
+// We add the ES5 adapter because src / test are converted to ES5 and native
```

Comment:
Comment is self-explanatory, but thought I should highlight this as part of the caveats listed in the main description of the PR.

Reply:
@WebReflection's ideas from [babel-plugin-transform-builtin-classes](https://github.com/WebReflection/babel-plugin-transform-builtin-classes/) have been incorporated [into Babel 7](https://github.com/babel/babel/pull/7020), so at some point this adapter will not be needed.

I already use WebComponents.js without native-shim in my projects, transpiling with Babel 6 + babel-plugin-transform-builtin-classes.

Reply:
The result of using babel-plugin-transform-builtin-classes is nice because the transpiled code run in all browsers (old and new).